.github/pull-request-template.md

```diff
@@ -0,0 +1,31 @@
+#### What does this PR do?
```

Comment:
I think we will want a modified PR template for this repo, as this one doesn't include some things we likely want and has other stuff we don't need. @JPrevost can we chat offline about what we want here?